raspberry/main.py

```diff
@@ -38,11 +38,13 @@
               initial_beta_t=-9.81)
     # update_measurement_at_fixed_rate: if True then use time difference between current time and last measurement time to take a measurement
     #            if false then take a measurement instantly
-    tof = TimeOfFlightSensor(update_measurement_at_fixed_rate=False,
-                             median_filter_length=1,
-                             cache_altitude=True,
-                             use_outlier_detection=True,
-                             abs_outlier_diff_thres=500)
+    tof = EvoSensor(self,
+                 serial_path='/dev/ttyAMA2',
+                 baud=115200,
+                 window_length=3,
+                 data_processor=NoFilter(),
```

Comment:
A median filter with length of 3 would be better here I guess

Reply:
Ok, sir